### PR TITLE
OKTA-577944 Auth Policy API Doc Updates from PA team

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Auth Policy API dev doc updates from Product Acceleration team based on PATEAM-389 and PATEAM-390
- **Is this PR related to a Monolith release?** No

### Preview:

https://63f55d2d1f7f8c0a16b97fa9--reverent-murdock-829d24.netlify.app/docs/reference/api/policy/#verification-method-object

### Resolves:

* [OKTA-577944](https://oktainc.atlassian.net/browse/OKTA-577944)
* [OKTA-578000](https://oktainc.atlassian.net/browse/OKTA-578000)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/404908faabdbb5193613d02a525cd6f7c0104ac0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f65adfe7-1f54-4072-9807-c76b10e3213d) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.76af1e55-e059-4cec-9426-ed5d4beba4b5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/7855b5df19ceb479a995d60bf45975f587f0521e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 367 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8a78c33b-5272-4f20-8866-4a18773ee0a0) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e1320890-f5c6-4485-b340-6a7af19f8359?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/b9cc29d10f23d8fdf3d72179279aafea040e1db5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 367 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/712f1129-5e41-4d11-9fa2-c379f410a075) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e29ec989-d362-4b64-a83f-5547f7c4f4ce?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by vanngo-okta** [comment]
Dan Marma reviewed and approved (commented in Jira ticket).

**Review by antonioruberto-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -2187,6 +2187,69 @@ The number of Authenticator class constraints in each Constraint object must be
...
+        "methods": [
+          "duo"
+        ]
+      }
+    }
+  ]
+}
+```
+
+###### Single factor WebAuthN
```
Comment (danielmaharry-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:2211]:
```suggestion
###### Single factor WebAuthn
```

**Review by danielmaharry-okta — APPROVED** [review]
One nit but approving not to hold you up

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/a135d7f7139e617ba81c769e13d368ca636be2a1/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 72 | 367 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3f9bb605-22ec-44e4-803c-ec18d8b440be) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e6434da7-cbe0-4cd6-be3e-6741957fec2c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by kentonsmith-okta — APPROVED** [review]

**Pull request merged**